views/pages/home.ejs

```diff
@@ -5,7 +5,7 @@
 <script src="js/asideShowTimes.js" type="module"></script>
 
 <main>
-	<section class="carousel">
+	<section class="carousel" hidden="hidden">
```

Comment:
Why did you hide the carousel? 🤔

Reply:
Seams unnecessary to have it since it loads movies from the old database, but I can leave it in.

Reply:
Think of all the work David put into making it pretty! 😭

Reply:
Well I put months of hard work in to our 404 page but you removed that :cry:

Reply:
We had a 404 page? You mean the placeholder page? It's still there! 🤔

Reply:
You are right, as usually, I mixed them up :flushed: